lib/start_game.rb

```diff
@@ -15,9 +18,24 @@ def initialize()
 
   def main_menu
     # Welcome to Battleship Functionality
+<<<<<<< HEAD
+    puts "Welcome to BATTLESHIP"
+    puts "Enter p to play. Enter q to quit."
+    user = gets.chomp
     # should end here if q
+    until user.downcase == "p"
+      return main_menu
+    end
```

Comment:
this loop is wrong. you want to make it as long as the input is not q it will continue the game